Makefile

```diff
@@ -23,13 +23,18 @@
 # A simple check to determine the tool is available. No version check, etc.
 REQUIRED_SOFTWARE=docker docker-compose git node yarn
 
-# List of projects that will be cloned and integrated into the system.
-# The project name must be in the ReactionCommerce Github org and the name must
-# match the Github project name.
-# Projects are started in order of this list.
-REACTION_PROJECTS=reaction-hydra \
-		  reaction \
-		  reaction-next-starterkit
+# Defined here are the subprojects in a comma-separated format
+# URL_PREFIX,REPO_NAME,TAG
+# URL_PREFIX is the start of a git repository URL (no trailing slash)
+# REPO_NAME is just the directory name itself
+# TAG is the git tag or branch to checkout
+# Projects will be started in this order
+define reaction-repos
+git@github.com:/reactioncommerce,reaction-hydra,master \
+git@github.com:/reactioncommerce,reaction,release-2.0.0-rc.2 \
```

Comment:
While we may want the latest API RC branch internally, `release-2.0.0-rc.2` should instead be `master` for the current stable release.

Reply:
Yeah these are just placeholders to illustrate the functionality we're aiming for (and I wanted to test that branches and tags worked)